scripts/build.py

```diff
@@ -163,3 +163,8 @@ def get_cache_slug():
     CSS_SRC_DIR = os.path.join('core', 'templates', 'prod', 'head', 'css', '')
     CSS_OUT_DIR = os.path.join(BUILD_DIR, 'css', '')
     copy_files_source_to_target(CSS_SRC_DIR, CSS_OUT_DIR)
+
+    # Copy core/templates/prod/head/ to build/[cache_slug]/templates/head/
+    TEMPLATES_SRC_DIR = os.path.join('core', 'templates', 'prod', 'head', '')
+    TEMPLATES_OUT_DIR = os.path.join(BUILD_DIR, 'templates', 'head', '')
```

Comment:
OK, so what you're saying is: move non-template files directly to build/, and keep html templates in /prod/head? That sounds reasonable until we're able to make the html templates static as well.

I defer to @BenHenning on whether to do this separately or not, since he has a better understanding of the remaining GSoC tasks than I do.